amplify/backend/function/fetchMotionState/src/utils.js

```diff
@@ -34,10 +33,8 @@ const getVotingClient = async (colonyAddress) => {
   return colonyClient.getExtensionClient(Extension.VotingReputation);
 };
 
-const getLatestMotionState = async (
-  colonyAddress,
-  { nativeMotionId, createdBy },
```

Comment:
Any reason for not destructuring the parameter?

Reply:
@willm30 When destructing directly in the parameter, I was getting issues with motionData being undefined.

Reply:
@Julianahlk  This isn't being caused by the destructuring, it's related to a refetch function that I have a fix for here:https://github.com/JoinColony/colonyCDapp/pull/505

Reply:
Got it, reverting 👍